ansible/playbooks/roles/compatibility_layer/tasks/install_prefix.yml

```diff
@@ -0,0 +1,56 @@
+# Install Gentoo Prefix.
+---
+- name: "Install EPEL"
+  yum:
+    name:
+      - https://dl.fedoraproject.org/pub/epel/epel-release-latest-8.noarch.rpm
+    disable_gpg_check: yes
+    state: present
+  tags:
+    - build_prefix
+
+- name: "Install other requirements (Singularity et al.)"
+  yum:
+    name:
+      - singularity
+    state: present
+  tags:
+    - build_prefix
+    
+- name: "Get the username running the deployment (not root)"
+  become: no
+  shell: set -o pipefail && who | grep -v root | awk '{print $1}' | uniq
+  register: username_on_host
+  changed_when: True
+  tags:
+    - build_prefix  
+
+- name: "Create Gentoo prefix path and log directory -- {{ gentoo_prefix_path }} and {{ eessi_log_dir }}"
+  file:
+    path: "{{ item }}"
+    state: directory
+    recurse: yes
+    owner: "{{ username_on_host.stdout }}"
+    group: "{{ username_on_host.stdout }}"
+    mode: "0755"
+  with_items:
+    - "{{ gentoo_prefix_path }}"
+    - "{{ eessi_log_dir }}"
+  tags:
+    - build_prefix
+
+- name: "Give {{ username_on_host.stdout }} recursive user and group ownership of /cvmfs"
+  file:
+    dest: "/cvmfs"
```

Comment:
Shouldn't this be just `{{ gentoo_prefix_path }}`? (and also in the title)

Reply:
Well... I saw something weird happen with permissions on /cvmfs itself. I haven't been able to reproduce it though.

Reply:
Right, okay, so we need a become:yes for our overlay due to issues with /proc, which is what causes this follow-up issue.

Reply:
and I'm testing this with the following:
```
- name: "Give {{ username_on_host.stdout }} recursive user and group ownership of {{ gentoo_prefix_path }}"
  file:
    dest: "{{ gentoo_prefix_path }}"
    owner: "{{ username_on_host.stdout }}"
    group: "{{ username_on_host.stdout }}"
    recurse: yes
  tags:
    - build_prefix
```